configs/base.js

```diff
@@ -27,7 +27,7 @@ export const baseConfig = {
         ...stylisticRuleSet.plugins,
         ...bestPracticesRuleSet.plugins,
         import: importPlugin,
-        'eslint-comments': eslintCommentsPlugin,
+        '@eslint-community/eslint-plugin-eslint-comments': eslintCommentsPlugin,
```

Comment:
1. The `eslint-plugin-` prefix was optional in the past in the configuration so `@eslint-community/eslint-comments` is equivalent to `@eslint-community/eslint-plugin-eslint-comments`
2. Since the introduction of flat configs we are completely free to decide the namespace on our own. I think I would prefer re-using the old name.

Reply:
🆗 ✅